v3/jaykeeb/aumz_work/aumsworksoldered.json

```diff
@@ -0,0 +1,223 @@
+{
+  "name": "Aums Work Soldered Version",
+  "vendorId": "0x414C",
+  "productId": "0x0772",
+  "matrix": {"rows": 5, "cols": 15},
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      "ISO Enter",
+      "Split LShift",
+      ["Bottom Row", "7U", "6.25U"]
+    ],
+    "keymap": [
```

Comment:
![image](https://github.com/user-attachments/assets/f95c71d2-b5e8-42b7-98a0-452ec36dd40c)

Key is missing group

Reply:
done